snappy/click.go

```diff
@@ -154,7 +166,6 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
 // verifyStructStringsAgainstWhitelist takes a struct and ensures that
 // the given whitelist regexp matches all string fields of the struct
 func verifyStructStringsAgainstWhitelist(s interface{}, whitelist *regexp.Regexp) error {
```

Comment:
The magic is deep on this one. :-)  Seems like explicitly validating the few fields we care about would be straightforward and leave less legroom for surprises. But I guess it doesn't matter much now, since click.go should be going away soon anyway.